src/app/organizations/manage/policy-edit.component.ts

```diff
@@ -172,6 +172,11 @@ export class PolicyEditComponent implements OnInit {
         }
     }
 
+    getCheckboxDesc(): string {
+        return this.type === PolicyType.PersonalOwnership ? this.i18nService.t('personalOwnershipCheckboxDesc') :
+            this.i18nService.t('enabled');
+    }
+
```

Comment:
This is just syntax sugar, so feel free to disregard, but this reads better as a getter to me.

```suggestion
    get checkboxDesc(): string {
        return this.type === PolicyType.PersonalOwnership ? this.i18nService.t('personalOwnershipCheckboxDesc') :
            this.i18nService.t('enabled');
    }

```

Reply:
Updated!